src/servers/ws-server.ts

```diff
@@ -0,0 +1,175 @@
+import { getOperationAST, GraphQLError, parse, validate } from "graphql"
+import { WebSocketServer } from "ws" // yarn add ws
+// import ws from 'ws'; yarn add ws@7
+// const WebSocketServer = ws.Server;
+import { gqlMainSchema } from "@graphql/main"
+import { useServer } from "graphql-ws/lib/use/ws"
+
+import { getJwksArgs, isDev } from "@config"
+import { Context } from "graphql-ws"
+import jsonwebtoken from "jsonwebtoken"
+
+import { parseIps } from "@domain/accounts-ips"
+
+import jwksRsa from "jwks-rsa"
+
+import { sendOathkeeperRequest } from "@services/oathkeeper"
+
+import { setupMongoConnection } from "@services/mongodb"
+
+import { baseLogger } from "@services/logger"
+
+import { sessionContext } from "./graphql-server"
+
+const schema = gqlMainSchema
+
+const port = 4000
```

Comment:
should we make this configurable?

Reply:
Yup. Feel free to take over this Pr if you'd like @ntheile 

I can do it but if you want a short term task this is probably a  good one.

Reply:
yeah i can work on it!